docs/UserGuide.md

```diff
@@ -3,7 +3,7 @@ layout: page
 title: User Guide
 ---
 * Table of Contents
-{:toc}
+  {:toc}
```

Comment:
Don't touch this line, it is not meant to be indented.

Reply:
OK, will change it. It's autocorrected by intellij and I didn't realise.